lib/battle/Unit.cpp

```diff
@@ -59,7 +59,7 @@ std::vector<BattleHex> Unit::getSurroundingHexes(BattleHex position, bool twoHex
 
 		if(side == BattleSide::ATTACKER)
 		{
-			for(BattleHex::EDir dir = BattleHex::EDir(0); dir <= BattleHex::EDir(4); dir = BattleHex::EDir(dir+1))
+			for(auto dir = static_cast<BattleHex::EDir>(0); dir <= static_cast<BattleHex::EDir>(4); dir = static_cast<BattleHex::EDir>(dir + 1))
```

Comment:
Interesting why it is 4 now and 5 in different places. 5 I understand: 6 directions starting from 0, but 4?

Reply:
It is because 2-hex unit so one hex it occupies and we need to skip it. Such logic is better to show explicitly by if(hex == otherHex) continue; as for me. But for sure not in scope of this PR).

Reply:
@nullkiller because it is for double-wide units. So, one direction is skipped